agave/models/helpers.py

```diff
@@ -1,9 +1,49 @@
+import datetime as dt
 import uuid
 from base64 import urlsafe_b64encode
+from typing import Any
+
+from rom import Column, Model, PrimaryKey
```

Comment:
You can't have this here. It'll break if running on a server without redis

Reply:
Done!